src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -3787,4 +3787,19 @@
   <data name="Arg_MemberInfoNotFound" xml:space="preserve">
     <value>A MemberInfo that matches '{0}' could not be found.</value>
   </data>
+  <data name="InvalidOperation_BadTZHeader" xml:space="preserve">
+    <value>Bad magic in '{0}': Header starts with '{1}' instead of 'tzdata'</value>
+  </data>
+  <data name="InvalidOperation_BadBuffer" xml:space="preserve">
+    <value>private error: buffer too small</value>
+  </data>
+  <data name="InvalidOperation_ReadTZError" xml:space="preserve">
+    <value>Unable to fully read from file '{0}' at offset {1} length {2}; read {3} bytes expected {4}.</value>
+  </data>
+  <data name="InvalidOperation_BadIndexLength" xml:space="preserve">
+    <value>Length in index file less than AndroidTzDataHeader</value>
+  </data>
+  <data name="InvalidOperation_TimeZoneIDNotFound" xml:space="preserve">
```

Comment:
Can this instead reuse TimeZoneNotFound_MissingData?

Reply:
Good catch, thanks!